libraries/AP_GPS/GPS_Backend.cpp

```diff
@@ -184,3 +184,43 @@ bool AP_GPS_Backend::should_df_log() const
     }
     return true;
 }
+
+
+void AP_GPS_Backend::send_mavlink_gps_rtk(mavlink_channel_t chan)
+{
+    const uint8_t instance = state.instance;
+    // send status
+    if (instance == 0) {
```

Comment:
Please use a switch instead, there is a bit of a long term desire to support more then just 2 GPS instances, which will result in all the additional GPS's being reported as gps2_rtk.